examples/nerdctl-ipfs-registry-kubernetes/bootstrap.yaml.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+#   Copyright The containerd Authors.
+
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+
+#       http://www.apache.org/licenses/LICENSE-2.0
+
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+# Example script to prepare swarm key secret for IPFS bootstrap,
+# Example: ./bootstrap.yaml.sh > ./bootstrap.yaml
+
+set -eu -o pipefail
+
+for d in ipfs-swarm-key-gen ; do
+    if ! which $d >/dev/null 2>&1 ; then
```

Comment:
Can we use `command -v`?

`which` is often missing.

Reply:
fixed